pkgs/top-level/all-packages.nix

```diff
@@ -13201,7 +13201,10 @@ with pkgs;
 
   tinyfecvpn = callPackage ../tools/networking/tinyfecvpn { };
 
-  tinygltf = callPackage ../development/libraries/tinygltf { };
+  tinygltf = throw ''
```

Comment:
Put this in `top-level/aliases.nix` .

Reply:
Isn't this still open, @AndersonTorres what changed your mind?

Reply:
Nothing, it is just there because someone forgot to hit the button.